docs/features/traffic-management/openshift.md

```diff
@@ -0,0 +1,61 @@
+# Openshift Routes
+
+[Openshift Routes](https://docs.openshift.com/container-platform/4.7/networking/routes/route-configuration.html) allow services to be exposed through externally-reachable hostnames. Openshift routes have additional functionality with traffic splitting between different services, allowing Argo Rollouts to shift traffic between different versions during a Canary deployment.
```

Comment:
better use latest version of the docs
https://docs.openshift.com/container-platform/4.11/networking/routes/route-configuration.html

Reply:
Thanks 👍